cinema/urls.py

```diff
@@ -18,6 +20,7 @@
 router.register("movie_sessions", MovieSessionViewSet)
 router.register("orders", OrderViewSet)
 
-urlpatterns = [path("", include(router.urls))]
+urlpatterns = ([path("", include(router.urls))]
+               + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT))
```

Comment:
The `urlpatterns` should be defined as a list, not a tuple. Remove the parentheses to ensure it is a list: `urlpatterns = [path('', include(router.urls))] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)`.